oak/server/rust/src/asylo_alloc.rs

```diff
@@ -0,0 +1,34 @@
+use core::alloc::{GlobalAlloc, Layout};
+
+extern "C" {
+    // See https://www.sourceware.org/newlib/libc.html#malloc.
+    fn malloc(nbytes: usize) -> *mut u8;
+    fn memalign(align: usize, nbytes: usize) -> *mut u8;
+    fn free(aptr: *mut u8);
+}
+
+// Aligment is defined in the Intel SGX SDK.
```

Comment:
Could you expand "alignment"?   (I think this is "The underlying Intel SGX SDK allocator will always return memory that is has at least this alignment (i.e. the address is a multiple of this)").